src/libdredd/include/libdredd/mutation_replace_expr.h

```diff
@@ -32,14 +32,18 @@ class MutationReplaceExpr : public Mutation {
 
   void Apply(
       clang::ASTContext& ast_context, const clang::Preprocessor& preprocessor,
-      int first_mutation_id_in_file, int& mutation_id, bool optimise_mutations,
+      bool optimise_mutations, int first_mutation_id_in_file, int& mutation_id,
       clang::Rewriter& rewriter,
       std::unordered_set<std::string>& dredd_declarations) const override;
 
   static void ApplyCppTypeModifiers(const clang::Expr* expr, std::string& type);
 
   static void ApplyCTypeModifiers(const clang::Expr* expr, std::string& type);
 
+  // Check if an expression is equivalent to a constant.
+  static bool ExprIsEquivalentTo(clang::Expr* expr, int x,
```

Comment:
Could `expr` be changed to `const clang::Expr& expr`?

Please change `x` to have a more descriptive name (both in the .h and .cc file).

Reply:
Changed both.